os/include/net/lwip/def.h

```diff
@@ -1,6 +1,6 @@
 /****************************************************************************
  *
- * Copyright 2016 Samsung Electronics All Rights Reserved.
+ * Copyright 2018 Samsung Electronics All Rights Reserved.
```

Comment:
Please check all of modified file. When you add new file, 2018 is correct. But if you modify existed file, previous year is correct. Because that shows when it was made (added).